bbq/infer_schema.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Copied from google? Please add the source URL where you got this file.

Reply:
That file is from Google :) from the BQ repo